src/Config/ConfigWriter.php

```diff
@@ -84,16 +84,27 @@ protected function parseContentValue($contents, $path, $value)
 
         $count = 0;
         $patterns = array();
+
+        // this should be first
+        $patterns[] = $this->buildEnvExpression($key, $items, false);
+        $patterns[] = $this->buildEnvExpression($key, $items);
+
         $patterns[] = $this->buildStringExpression($key, $items);
         $patterns[] = $this->buildStringExpression($key, $items, '"');
         $patterns[] = $this->buildConstantExpression($key, $items);
         $patterns[] = $this->buildArrayExpression($key, $items);
 
         foreach ($patterns as $pattern) {
-            $result = preg_replace($pattern, '${1}${2}'.$replaceValue, $result, 1, $count);
-
-            if ($count > 0) {
-                break;
+            if ($pattern === $patterns[0]) {
```

Comment:
Why has this been added? What does it do?

Reply:
for match  `'KEY' = > env('KEY')`,because the `$result = preg_replace($pattern, '${1}${2}' . $replaceValue, $result, 1, $count);` will lead the replace result to `KEY' = > env('KEY'`. the `)` will missing. so here added another rule of `$result = preg_replace($pattern, '${1}${2}' . ',' . $replaceValue . ')', $result, 1, $count);`
```

```